src/rooms/room_members/v1.rs

```diff
@@ -0,0 +1,38 @@
+//! [GET /_synapse/admin/v1/rooms/:room_id/members](https://github.com/element-hq/synapse/blob/master/docs/admin_api/rooms.md#room-members-api)
+use ruma::{
+    api::{metadata, request, response, Metadata},
+    OwnedRoomId, OwnedUserId, UInt,
+};
+
+const METADATA: Metadata = metadata! {
+    method: GET,
+    rate_limited: false,
+    authentication: AccessToken,
+    history: {
+        unstable => "/_synapse/admin/v1/rooms/:room_id/members",
+    }
+};
+
+#[request]
+pub struct Request {
+    /// Alias or ID of the room to join.
+    #[ruma_api(path)]
+    pub room_id: OwnedRoomId,
+}
+
+#[response]
+pub struct Response {
+    /// Room ID
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
```

Comment:
This attribute doesn't seem necessary, according to [the code](https://github.com/element-hq/synapse/blob/5c736cd2af3eb9e1264d439c274d608f6ffc6a83/synapse/rest/admin/rooms.py#L438) this field is always present in the response.